### PR TITLE
Adding imbues to damage formulas: Critical Strike, Crippling Blow, Armor Rending, Resistance Rending

### PR DESCRIPTION
**Pull request by gmriggs**

### CODE REVIEW

File: Source/ACE.Server/WorldObjects/Player_Inventory.cs [hunk 1]
```diff
@@ -887,8 +887,13 @@ private bool DoHandleActionGetAndWieldItem(WorldObject item, Container itemRootO
             return true;
         }
 
+        public static bool UseWieldRequirements = true;
```
Comment (Mag-nus) [inline Source/ACE.Server/WorldObjects/Player_Inventory.cs]: This can't be static, it will break if work is done on multiple threads.

Comment (Mag-nus) [inline Source/ACE.Server/WorldObjects/Player_Inventory.cs]: This can't be static, it will break if work is done on multiple threads.

Comment (Mag-nus) [reply Source/ACE.Server/WorldObjects/Player_Inventory.cs]: Follow up question, where is this set to false?

Comment (gmriggs) [reply Source/ACE.Server/WorldObjects/Player_Inventory.cs]: this is an internal debugging only var

Comment (Mag-nus) [reply Source/ACE.Server/WorldObjects/Player_Inventory.cs]: ahh, ok. A comment on it would be good. I saw it before during the refactor and was confused.

**Pull request merged**